src/creature.cpp

```diff
@@ -102,6 +103,24 @@ void Creature::reset()
     reset_bonuses();
     reset_stats();
 }
+
+void Creature::bleed( const tripoint &where, int intensity ) const
```

Comment:
I don't really see a good reason for this function to exist in `Creature`. It delegates bleeding on map to the critter and re-gets a lot of stats over and over.

It should be blood splatter function's role to do the bleeding, critter should only provide blood type. Hallucinations can be handled by giving them no blood type.